### DISCUSSION TITLE
rebind hotkeys?

### DISCUSSION THREAD

**Discussion by Float-pixel** [jserodio/rg351p-ppsspp-settings#11] [Q&A - Drop your questions here]
I am using PPSSPP on the RG351M using ArkOs 1.6

I cannot figure out how to rebind the emulator hotkeys. Things like L2 opening menu, Select + B opening menu, Select + X/Y loading or saving savestates.

Can't find a way to change these in the emulator menu or through the .ini files.

Any ideas?

**Comment by jserodio** [comment]
I'm sorry but I don't think this is the right place to ask about this.
Consider dropping your question here instead
https://discord.gg/tsRyxZzP
